lib/message.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+
+class Message
+  # Note that I'm using test credentials for sid, auth and from_no
+  # As far as I can tell, there's no way of using live SMS without paying
+
+  def initialize
+    @account_sid = ENV['TWILIO_ACCOUNT_SID']
+    @auth_token = ENV['TWILIO_AUTH_TOKEN']
+    @sender = ENV['MY_FROM_NUMBER']
+  end
+
+  def send(customer, time)
```

Comment:
This could be refactored into 4 methods: this one; one to calculate the time, one to create the Twilio client (maybe a reader?) and one to create the Twilio message